Cython/Utility/ModuleSetupCode.c

```diff
@@ -564,6 +564,13 @@ class __Pyx_FakeReference {
   #define __Pyx_PyVectorcall_NARGS(n)  (n)
 #endif
 
+// PEP-573
+#if PY_VERSION_HEX >= 0x030900B1
+  #define __PyType_FromModuleAndSpec(m, s, b)  PyType_FromModuleAndSpec(m, s, b)
```

Comment:
Just wondering, why isn't this `__Pyx_PyType_FromModuleAndSpec`?

Reply:
You seem to be referring to old revisions of the code here.